sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -747,7 +747,7 @@ class CudfRegexTranspiler(mode: RegexMode) {
     e match {
       case RegexEscaped(ch) if escapeChars.contains(ch) => Some(escapeChars(ch).toString)
       case RegexEscaped(ch) if regexPunct.contains(ch) => Some(ch.toString)
-      case RegexChar(ch) if !regexMetaChars.contains(ch) => Some(ch.toString)
+      case RegexChar(ch) if regexMetaChars.contains(ch) => Some(ch.toString)
```

Comment:
Why is the negative condition removed here? This should be negated in this part.

Reply:
Thanks, done.